include/pybind11/pybind11.h

```diff
@@ -1939,6 +1951,18 @@ void register_exception_translator(ExceptionTranslator&& translator) {
         std::forward<ExceptionTranslator>(translator));
 }
 
+
+/**
+  * Add a new module-local exception translator. This function will be applied
```

Comment:
> "This function will be applied"

"These will be tried ..." or so? "applied" seems weird

Reply:
verbiage has been adjusted

Reply:
You can tell I've been doing a lot of functional programming lately, because I'm out here applying functions to things

Reply:
Hah :-D Thanks!